schema/cadence/schema.cql

```diff
@@ -317,3 +318,19 @@ CREATE TABLE domains_by_name (
 )  WITH COMPACTION = {
      'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
    };
+
+CREATE TABLE domains_by_name_v2 (
+  domains_partition             int,
+  name                          text,
+  domain                        frozen<domain>,
+  config                        frozen<domain_config>,
+  replication_config            frozen<domain_replication_config>, -- indicating active cluster and standby cluster used for replication
+  is_global_domain              boolean, -- indicating whether a domain is a global domain
+  config_version                bigint, -- indicating the version of domain config, excluding the failover / change of active cluster name
+  failover_version              bigint, -- indicating the version of active domain only, used for domain failover
+  failover_notification_version bigint, -- indicating the last change related to domain failover
```

Comment:
why do we need 2 separate notification versions?
Previously we used to have db_version too, why we don't need that anymore?

Reply:
the failover_notification_version is actually your idea, tracking the last change which related to domain failover

the previous db_version is now the notification_version, managed by the metadata record